src/main/java/seedu/address/logic/commands/AddCommand.java

```diff
@@ -27,7 +26,6 @@ public class AddCommand extends Command {
             + PREFIX_EMAIL + "EMAIL "
             + PREFIX_ADDRESS + "ADDRESS "
             + PREFIX_DOB + "DATE OF BIRTH "
-            + "[" + PREFIX_TAG + "TAG] "
             + "[" + PREFIX_GENDER + "GENDER] ... \n"
```

Comment:
Maybe we could remove the ... after gender since without tags there can't be any more parameters after gender
```suggestion
            + PREFIX_GENDER + "GENDER \n"
```